crates/bevy_ui/src/ui_node.rs

```diff
@@ -2266,6 +2266,39 @@ impl ResolvedBorderRadius {
     };
 }
 
+#[derive(Component, Copy, Clone, Debug, PartialEq, Reflect)]
+#[reflect(Component, PartialEq, Default)]
+#[cfg_attr(
+    feature = "serialize",
+    derive(serde::Serialize, serde::Deserialize),
+    reflect(Serialize, Deserialize)
+)]
+pub struct BoxShadow {
+    /// The shadow's color
+    pub color: Color,
+    /// Offset increasing to the right and down
```

Comment:
These doc comments need to be duplicated / split across the various fields, otherwise they won't show up on IDE support nicely.